onlyoffice-nextcloud/exports.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+export DOCSERVER_PORT=5672
+export APP_DOCSERVER_PORT=3014
+
+export DEFAULT_INTERFACE_IP=$(ip addr show $(ip route | grep default | awk '{print $5}') | grep 'inet ' | awk '{print $2}' | cut -d/ -f1)
+export DOCKER_INTERFACE_IP=$(ip addr show docker0 | grep 'inet ' | awk '{print $2}' | cut -d/ -f1)
```

Comment:
If we end up going with the simplifications here (), then we can remove these exports.

Reply:
To elaborate on the name change, it is a good idea and will improve the appearance of the app, but I also think some improvements can be made to the whole app in light of that.

I'm also consdering adding a startup screen, similar to how the Bitcoin Node and Lightning Node apps have it with a progress bar, so if a user clicks on the ONLYOFFICE Docs app, they will be redirected to the actual app within Nextcloud in a few seconds with the opportunity to cancel and view the setup instructions instead. 

Ideally, there would be some configuration detection mechanism so it doesn't redirect upon the very first launch and only when it knows Nextcloud has been configured, but I think that's for a future update for now.